R/deprecate.R

```diff
@@ -0,0 +1,40 @@
+#' These functions will be deprecated. Please use other functions instead.
+#' 
+#' @param x a \code{\link{SummarizedExperiment}} object -
+#' 
+#' @param ... -
+#' 
+#' @name deprecate
+NULL
+
+#' @rdname deprecate
+setGeneric("addTaxonomyTree",
+            signature = "x",
+            function(x, ...)
+                standardGeneric("addTaxonomyTree"))
+
+#' @rdname deprecate
+#' @export
+setMethod("addTaxonomyTree", signature = c(x = "SummarizedExperiment"),
+            function(x){
+                .Deprecated(msg = paste0("'addTaxonomyTree' is deprecated.\n",
+                                        "Use 'addHierarchyTree' instead."))
```

Comment:
Do not add new line ("\n"). It is tidier without (because there might be multiple warnings